SafeguardSessions/SafeguardSessions.proj

```diff
@@ -4,7 +4,7 @@
     <OutputPath Condition="'$(OutputPath)' == ''">$(MSBuildProjectDirectory)\bin\</OutputPath>
     <IntermediateOutputPath Condition="'$(IntermediateOutputPath)' == ''">$(MSBuildProjectDirectory)\obj\</IntermediateOutputPath>
     <MezIntermediatePath>$(IntermediateOutputPath)MEZ\</MezIntermediatePath>
-    <MezOutputPath>$(OutputPath)$(MsBuildProjectName).mez</MezOutputPath>
+    <MezOutputPath>$(OutputPath)OneIdentitySafeguard.mez</MezOutputPath>
```

Comment:
Please follow up this at:
https://github.com/OneIdentity/SafeguardPowerBI/blob/948b0aa412439ae720a58d029d92bb5a12b0a6af/SafeguardSessions/Makefile#L12

Reply:
Nice one!

Reply:
Thank you! I checked the tests are still working after renaming, however I forgot to delete the old SafeguardSessions.mez from the bin folder, so it gave me false positive feedback.